docs/content/concepts/partitions-schedules-sensors/partitioning-assets.mdx

```diff
@@ -216,6 +216,12 @@ If using the default I/O manager, materializing partition `2022-07-23` of this a
 
 ---
 
+## Recommended partition limits
+
+We recommended limiting partition counts for each asset to 25,000 partitions or fewer. Assets with partition counts exceeding this limit will likely have slower load times in the UI.
```

Comment:
```suggestion
We recommend limiting the number of partitions for each asset to 25,000 or fewer. Assets with partition counts exceeding this limit will likely have slower load times in the UI.
```

- `recommended` should be `recommend`
- Is the correct term actually `partition counts`?

Reply:
thanks -- yeah I think "number of partitions" is clearer